choco/src/it/ChocoWebIntegrationTest.groovy

```diff
@@ -0,0 +1,52 @@
+import java.nio.file.Paths
+import org.apache.commons.io.FileUtils
+
+class ChocoWebIntegrationTest
+{
+
+    def getExecutionBasePath(project)
+    {
+        return Paths.get(System.getProperty("user.dir")).resolve('src').resolve('it').resolve('common-flows')
```

Comment:
Why are you using `System.getProperty` instead of relying on `project.basedir` which you are already getting as a parameter? Also, this should be pointing to the `target` directory so the generated content is automatically cleared with `mvn clean install` :)